cirq-google/cirq_google/devices/grid_device.py

```diff
@@ -276,3 +381,94 @@ def _from_json_dict_(cls, metadata, **kwargs):
 
     def _value_equality_values_(self):
         return self._metadata
+
+
+def _set_gate_in_gate_spec(
+    gate_spec: v2.device_pb2.GateSpecification, gate_family: cirq.GateFamily
+) -> None:
+    if gate_family == cirq.GateFamily(ops.SYC):
```

Comment:
I am not sure if it matters too much for this serialization, but this would construct each gate family for each function call.  

I think it might make sense to make these constants, since you could also potentially use them above too.

Reply:
Ah true. To keep the code just a tad more readable (as opposed to having to follow constant definitions), I opted to use `gate_family.gate` instead of instantiating a new GateFamily where possible, and created constants for virtual and physical Zs.

Reply:
Updated. I left `_build_gateset_and_gate_durations` as is to keep it simple (GateFamily isn't required) and to keep the consistency between the `fsim_gates.append()` lines and `cirq_gates = ` lines, but no strong preference either way.